frontend/server/src/main/java/org/pytorch/serve/wlm/WorkerThread.java

```diff
@@ -249,11 +250,40 @@ public void run() {
             }
         } catch (WorkerInitializationException e) {
             logger.error("Backend worker error", e);
+            // TODO: Need more detailed management for ModelServer exception
+            // along with error message and error code mapping
+            if (java.lang.System.getenv("SM_TELEMETRY_LOG") != null) {
+                loggerTelemetryMetrics.info(
+                        "UserScriptError.Count:1|#TorchServe:{},{}:-1|#hostname:{},timestamp:{}",
+                        ConfigManager.getInstance().getVersion(),
+                        e.getClass().getCanonicalName(),
+                        ConfigManager.getInstance().getHostName(),
+                        String.valueOf(
+                                TimeUnit.MILLISECONDS.toSeconds(System.currentTimeMillis())));
+            }
         } catch (OutOfMemoryError oom) {
             logger.error("Out of memory error when creating workers", oom);
             status = HttpURLConnection.HTTP_ENTITY_TOO_LARGE;
+            if (java.lang.System.getenv("SM_TELEMETRY_LOG") != null) {
```

Comment:
here needs to check on or off

Reply:
Should I make any modifications now since we are choosing to perform a null check for the SM_TELEMETRY_LOG to enable/disable telemetry?